.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,55 @@
+---
+name: Feature request
+about: Suggest an idea for this project
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+## Description
+
+Provide a clear and concise description of the requested feature.
+
+## Problem to Solve
+
+Explain the problem that this feature is trying to solve or the use case that it is trying to support.
+
+## Proposed Solution
+
+Describe how you envision this feature being implemented.
+
+## Additional Context
+
+Add any other context or examples to help explain the requested feature, including:
+- Screenshots that illustrate the issue or desired outcome
+- Existing parts of the feature in Antares Simulator, if applicable
+
+## Possible Alternatives
+
+If you have considered alternative solutions to the problem, describe them here.
+
+## Impact and Priority
```

Comment:
For me "Impact and Priority" duplicates with labels and "Developer Notes" is useless